### ISSUE TITLE
Do not state that the upload succeeeded when an error happens

### ISSUE THREAD

**Issue by laurentsigal** [waldoapp/waldo-cli#7]
See example:
```
0.73s
$ $HOME/waldo $APP_APK_PATH
2615
Waldo CLI 1.3.0 (Linux)
2616
Uploading the build to Waldo. This could take a while…
2617
curl: (3) Illegal characters found in URL
2618
waldo: Build failed to upload to Waldo: 3
2619
Build 'app-mock-debug.apk' successfully uploaded to Waldo!
2620
The command "$HOME/waldo $APP_APK_PATH" exited with 0.
```
exit code 0 is great, but we should display a warning

**Comment by laurentsigal** [comment]
It should show:
```
The build could not upload because of "{error}" - our team received the error
```

**Comment by eBardX** [comment]
@laurentsigal Do you also want me to fix the URL encoding issue? Or should that continue too fail?

**Comment by eBardX** [comment]
Implemented in `1.4.0`.
